packages/fxa-react/configs/tailwind.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
         24: '6rem',
       },
       maxWidth: {
+        '1xs': '16rem',
```

Comment:
I think this should be `64: '16rem'`.

Reply:
Should it? I was going with the scale we already use as the Tailwind [default](https://tailwindcss.com/docs/max-width). I'm okay changing to 64, just want to make sure it's consistent

Reply:
Ah I see, I'm fine with that too. I was going by what we already have on [this line](https://github.com/mozilla/fxa/blob/41952909aa3fdd02861607d7f6e5be05a774739d/packages/fxa-react/configs/tailwind.js#L54), but that's much smaller than the defaults.

Reply:
Oh right, I see we've already started using the other named sizes (mobileLandscape, desktop) - I'll go with 64 here. Good catch